test/dlc_test.json

```diff
@@ -0,0 +1,329 @@
+[ {
+  "testInputs" : {
+    "params" : {
+      "oracleInfo" : {
+        "publicKey" : "a78bf5ed2841a93d4dcc1b719ebbcc90c7648746e5bd55fe7806494a93167c39",
+        "nonce" : "d2d6cb7049ef4edc2d98aa3e235e8d542573cf0c9a735516b9936aba52ae693e"
+      },
+      "contractInfo" : {
+        "8873e27fd5804a500792d3b197ec83e3d13311e161c431b223657b64ec78e355" : 0,
+        "9fc2710e301f8700a8a4c6933366c52045f23a59c92eaf17b0c3d6413861d068" : 200000000
+      },
+      "contractMaturityBound" : 100,
+      "contractTimeout" : 200,
+      "feeRate" : 5
+    },
+    "offerParams" : {
+      "collateral" : 100000000,
+      "fundingInputTxs" : [ {
```

Comment:
Shouldn't you include the input value as well?

Reply:
Just realized that `tx` is a transaction and not a txid (which I had assumed). Is there a specific reason not to just pass a txid and value since that is what is needed to build the messages and txs?

Reply:
And rust-bitcoin seem not to be able to parse this transaction, AFAICT due to [how they resolve ambiguity of zero input transactions](https://github.com/rust-bitcoin/rust-bitcoin/blob/master/src/blockdata/transaction.rs#L243). So I would really recommend using txid + value to make things simpler.

Reply:
Sadly this is not possible: https://github.com/niftynei/lightning-rfc/pull/1#discussion_r471762248 (at least until we have Taproot)

Reply:
But I can add a dummy input if that would make your life easier

Reply:
added

Reply:
Thanks (haven't checked yet but hopefully it should work :) )